03-charts/README.md

```diff
@@ -140,7 +140,7 @@ And also change one thing in `service`:
 
 (See the images list in the main [README](../README.md) for a quick description of each.)
 
-Once you've done that, you will have a runnable chart that we can test. We'll be able to install it, but not use it.
+Once you've done that, you will have a runnable chart that we can test. We'll be able to install it, but it will not fully work, until $the_rest_of_the_stuff
```

Comment:
I think it might help to have more guidance about what they should expect to see and when it will have enough components to work, something like "we're now at step 2, and at step 3 we'll expect it to work fully; right now we're still missing (thing we're about to set up).